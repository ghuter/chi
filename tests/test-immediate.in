0123
123
1.23
0x123
0XABC
0Xabc
0b111
0B101
0xabcz
123xx
