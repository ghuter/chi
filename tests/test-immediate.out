IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
NEWLINE(4)
IMMEDIATE(2)
IDENTIFIER(3)
NEWLINE(4)
IMMEDIATE(2)
IDENTIFIER(3)
EOI(1)
